docs/dev_guide/contents/documentation.rst

```diff
@@ -109,8 +109,6 @@ In addition, a reference section must be provided with links to the following re
 * information to interpret metadata keywords such as FITS header reference
 * the data archive
 
-An example docstring can be found in the :ref:`Writing a new Instrument Map Class guide <new_maps_ts_etc>`.
```

Comment:
Is it worth leaving this sentence in and moving the link?

Reply:
How about if I link to Generic Maps docstring instead?

Reply:
The GenericMap docstring is not an example of a source docstring though.

Reply:
Added a sentence.